randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "the grate above can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3344095433": [
+        # Ruined Shrine PB Sandstone Tip
+        # Replaces Ruined Shrine sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3485565452": [
+        # Vault PB Slots Tip
+        # Replaces Vault bomb slot seal scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;These Bomb Slot covers are able to be destroyed by &main-color=#FF3333;Power Bombs"
+            "&main-color=#89D6FF;.\n\nAdditionally, these Bomb Slots are also able to be activated by Power Bombs."
+        ),
+    ],
+    "1354393071": [
+        # Watery Hall PB Sandstone Tip
+        # Replaces Watery Hall sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3507547160": [
+        # Lava Lake PB Sandstone + No-Respawn Tip
+        # Replaces Lava Lake sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;.\n\n"
+            "Additionally, in most game versions, these rocks will not reappear after being destroyed."
+        ),
+    ],
+    "3269646811": [
+        # Reflecting Pool PB Talloric Alloy Tip
+        # Replaces Reflecting Pool lid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "1876916293": [
+        # Fungal Hall B PB Tip
+        # Replaces MQB lower right computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item within Fungal Hall B's mushroom ring can be uncovered with "
+            "&main-color=#FF3333;Morph Ball Bombs&main-color=#89D6FF; or "
+            "&main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
```

Comment:
This isn't a trick, and players likely already know that this can be destroyed with power bombs and morph ball bombs. Additionally, this scan node provides information for a different room. This is superfluous.

Reply:
I've experienced beginners assuming it had to be PBs and being surprised when normal bombs work. I wouldn't have added this otherwise.